deepchem/feat/molecule_featurizers/molgan_featurizer.py

```diff
@@ -0,0 +1,214 @@
+import logging
+import numpy as np
+import rdkit.Chem as Chem
+
+from deepchem.utils.typing import RDKitBond, RDKitMol, List
+from deepchem.feat.base_classes import MolecularFeaturizer
+
+logger = logging.getLogger(__name__)
+
+
+class GraphMatrix:
```

Comment:
It looks like you're using 4 space spacing. DeepChem defaults to 2-space spacing. Could you swap to 2-space defaults? You can do this by placing the following lines in your `setup.cfg`

```
[yapf]
based_on_style = google
indent_width = 2
```

Reply:
I am just wondering, do you think this class should be placed in a different place?